test_regress/t/t_split_var_0.pl

```diff
@@ -14,7 +14,7 @@
 # %Warning-UNOPTTHREADS: Thread scheduler is unable to provide requested parallelism; consider asking for fewer threads.
 # So use 6 threads here though it's not optimal in performace wise, but ok.
 compile(
-    verilator_flags2 => ['--stats' . ($Self->{vltmt} ? ' --threads 6' : '')],
+    verilator_flags2 => ['--stats' . ($Self->{vltmt} ? ' --threads 6' : '') . " $Self->{t_dir}/t_split_var_0.vlt"],
```

Comment:
Use a comma, then it will be a new argument
     verilator_flags2 => ['--stats' . ($Self->{vltmt} ? ' --threads 6' : ''),
                                    "$Self->{t_dir}/t_split_var_0.vlt"],